### ISSUE TITLE
Add support for CtrlP as FZF

### ISSUE THREAD

**Issue by juanolon** [autozimu/LanguageClient-neovim#681]
## Is your feature request related to a problem? Please describe.
nop

## Describe the solution you'd like
Support for CtrlP as an alternative for FZF would be great. CtrlP also offers an extension interface (https://github.com/kien/ctrlp.vim/tree/extensions)
 
## Describe alternatives you've considered
I tried to use https://github.com/DeaR/ctrlp-location-list to show the list of references/implementations like this:
```
nnoremap <silent> <Leader>sr :call LanguageClient#textDocument_references()<CR> <bar> :CtrlPLocListAll<CR>
```
But CtrlP doesnt get the results as (i think) they are async. After locList get the results, it's possible to see them on CtrlPLocListAll

**Comment by nkakouros** [comment]
I tried https://github.com/DeaR/ctrlp-location-list with neovim and it doesn't seem to work at all.

**Comment by martskins** [comment]
This is probably now solved by #1059 (provided ctrlp-location-list works I assume). In any case, I'm gonna close this one based on the assumption that that PR enables this, but feel free to re-open if it still doesn't work.
